src/components/native-context-menu/contextMenu.android.js

```diff
@@ -7,6 +7,7 @@ export default function ContextMenuAndroid({
   menuConfig: { menuItems, menuTitle },
   isAnchoredToRight,
   onPressMenuItem,
+  shouldOpenOnLongPress,
```

Comment:
This is not needed for this PR anymore, but it is supported by the lib so it might make some sense to leave it to highlight to devs that this is available. Not sure though.